packages/mui-joy/src/AccordionSummary/AccordionSummary.tsx

```diff
@@ -148,6 +150,7 @@ const AccordionSummary = React.forwardRef(function AccordionSummary(inProps, ref
       'aria-expanded': expanded ? 'true' : 'false',
       'aria-controls': `${accordionId}-details`,
       disabled,
+      type,
```

Comment:
```suggestion
      type: 'button',
```

I think this PR requires only this line change.

Reply:
comment resolved